Misha/Files/pdf_converter/pdf_converter_main.py

```diff
@@ -0,0 +1,50 @@
+from fpdf import FPDF
+
+
+def read_from_file(path):
+    try:
+        f = open(str(path), 'r')
+        text = ''
+        for line in f:
+            text += line
+        f.close()
+        return text
+    except FileNotFoundError:
+        raise FileNotFoundError('No such file or directory')
+
+
+def change_format_to_pdf(path):
```

Comment:
I don't think that you need to change file extension manually... You shouldn't do it at all